lib/workload/stateless/universal-event-archiver/function/achiver.py

```diff
@@ -0,0 +1,43 @@
+import json
+import boto3
+from datetime import datetime, timezone
+import os
+
+# Initialize S3 client
+s3 = boto3.client('s3')
+
+# The name of the bucket
+BUCKET_NAME = os.getenv('BUCKET_NAME')
+
+def handler(event, context):
+    
+    # Current timestamp
+    now = datetime.now(timezone.utc)
+    time_stamp = now.strftime("%H_%M_%S") # for object name
```

Comment:
What if we have two or more events within the same second? 
Perhaps just use the `timestamp` ?

Reply:
sure, will use the timestamp directly instead.